deploy.sh

```diff
@@ -15,17 +15,36 @@ check_dep npm
 check_dep parity
 check_dep python3
 check_dep pip3
-check_dep mongod
 
+ROOT=$(pwd)
 ENERGY_MIN=-100
 ENERGY_MAX=100
-PROJ_DIR=$(pwd)
-cd $PROJ_DIR/truffle && rm -rf build && truffle compile && truffle build
-cd $PROJ_DIR/dapp && npm install && webpack && ./add_to_parity.sh
-cd $PROJ_DIR/parity && ./run.sh &
-cd $PROJ_DIR && pip3 install --user -r requirements.txt
-cd $PROJ_DIR/client/meter && npm install && npm start &
-cd $PROJ_DIR/simulation && python3 meter.py -e $ENERGY_MIN $ENERGY_MAX &
+
+if [ "$1" != "--no-truffle" ] && [ "$2" != "--no-truffle" ]
```

Comment:
I don't really understand why we need those flags, but i'm fine with that.

Reply:
To be honest, at first when I was running everything with deploy.sh, it was very useful. I soon realised that running deploy.sh while developing is useless as I cant control each individual server. So this PR is pretty redundant now